Animations/Animation.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+
+var React = require("react/addons");
+var Arrival = require('Arrival');
+var Easings = require("./AnimationEasings.js");
+var ReactTransitionGroup = React.addons.TransitionGroup;
+
+//makeTransition FUNCTION: creates animation for wrapped component based on props of wrapper component
+  //@param displayName STRING: intended name of the wrapper animation component
+  //@param transitions OBJECT: properties that define the default animation properties for the animation component wrapper
+var makeTransition = function (displayName, transitions) {
+
+  // setAnimationDomNode FUNCTION: function to handle manipulating and setting a TransitionGroup in the DOM.
+    //@param mount STRING: the transition key you are looking to affect
+    //@param done: done attribute passed from the containing function
```

Comment:
What is 'done'? If this is a callback recommend renaming it and updating comments or updating comments to reflect that information.